src/router-view.js

```diff
@@ -37,17 +37,23 @@ const swapStrategies = new SwapStrategies();
 
 @customElement('router-view')
 @noView
-@inject(DOM.Element, Container, ViewSlot, Router, ViewLocator, CompositionTransaction)
+@inject(DOM.Element, Container, ViewSlot, Router, ViewLocator, CompositionTransaction, CompositionEngine)
 export class RouterView {
   @bindable swapOrder;
+  @bindable layout;
+  @bindable layoutView;
```

Comment:
@alvarezmario I actually took the conscious decision to use `layout` because I didn't like the caps in `layoutView` :)

We actually need `layoutViewModel` and `layoutView` (one for the module Id one for the view - neither are required or mutual). You can also use `layoutModel` too - on the element they would obviously be lower-case hyphenated.

So basically - I agree, it should be consistently named